doc/ug/getting_started_dv.md

```diff
@@ -88,3 +88,23 @@ Please refer to the [checklist]({{< relref "doc/project/checklist" >}}) to under
 
 The [UART DV](https://github.com/lowRISC/opentitan/tree/master/hw/ip/uart/dv) area can be used as a canonical example for making progress.
 If it is not clear on how to proceed, feel free to file an issue requesting assistance.
+
+## Reproduce DV failure in CI(private)
+
+Follow these steps to reproduce CI(private) failure
+
+1. Make sure the version of VCS is the same as the [one](https://github.com/lowRISC/opentitan-private-ci/blob/master/jobs.yml#L5) running in CI
+
+2. CI runs against an auto-generated merge commit, which effectively is the PR merge into master.
```

Comment:
```suggestion
2. CI runs against an auto-generated merge commit, which effectively is the pull request (PR) merged into master.
```

Reply:
As Sri said, or maybe this would be clearer as something like "..., which is generated by merging the PR into the master branch"?